.gitignore

```diff
@@ -15,6 +15,7 @@ Build/Visual Studio 2017/Debug
 
 # cmake
 out
+build
```

Comment:
and also a comment here to explain that this is the build dir for linux as per HACKING